src/buildercore/terraform.py

```diff
@@ -1454,20 +1486,78 @@ def init(stackname, context):
             },
         }
 
+    backend = {
+        'terraform': {
+            # 'required_providers': {
+            #    'fastly': {
+            #        'source': 'fastly/fastly',
+            #    }
+            # },
+            'backend': {
+                's3': {
+                    'bucket': config.BUILDER_BUCKET,
+                    'key': 'terraform/%s.tfstate' % stackname,
+                    'region': config.BUILDER_REGION,
+                },
+            },
+        },
+    }
+
+    # 0.11.x => 0.13 transformations
+    # TODO: integrate section after Fastly projects updated to Terraform 0.13
+    if context['terraform']['version'] == '0.13.7':
+        # 'providers' now relies on a 'required_providers' block under 'terraform'
+        # - https://developer.hashicorp.com/terraform/language/v1.1.x/providers/requirements
```

Comment:
apparently not if @scottaubrey was able to upgrade his EKS projects without one. Investigate.